assets/airports/lrop.json

```diff
@@ -0,0 +1,674 @@
+            "altitude": [11000, 14000],
+            "speed": 250,
+            "method": "wave",
+            "rate": 15,
+            "airlines": [
+                ["ROT", 5],
+				["RYR", 4],
+				["DLH", 3],
+				["KLM", 2],
+				["ASL", 1],
+				["AFR", 2],
+				["BAW", 1],
+				["EWG", 1],
+				["LOT", 1],
+				["SWR", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "KOMAN.OBUGA1U.LROP",
+            "altitude": [14000, 17000],
+            "speed": 250,
+            "method": "random",
+            "rate": 5,
+            "airlines": [
+                ["ROT", 3],
+                ["THY", 2],
+				["PGT", 1],
+				["QTR", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "ARGES.OSTAL1U.LROP",
+            "altitude":  [15000, 17000],
+            "speed": 350,
+            "method": "random",
+            "rate": 5,
+            "airlines": [
+                ["ROT", 3],
+                ["THY", 2],
+				["PGT", 1],
+				["QTR", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "DENAK.DENAK1U.LROP",
+            "altitude":  [15000, 17000],
+            "speed": 350,
+            "method": "random",
+            "rate": 5,
+            "airlines": [
+                ["ROT", 5],
+				["RYR", 4],
+				["CSA", 1],
+				["LOT", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "NETUL.NETUL2U.LROP",
+            "altitude":  [17000, 20000],
+            "speed": 350,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["ROT", 1]
+				
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "IDARU.IDARU2U.LROP",
+            "altitude":  [17000, 20000],
+            "speed": 350,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["ROT", 1]
+				
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "DENAK.DENAK1U.LROP",
+            "altitude":  [15000, 17000],
+            "speed": 350,
+            "method": "random",
+            "rate": 5,
+            "airlines": [
+                ["ROT", 5],
+				["RYR", 4],
+				["CSA", 1],
+				["LOT", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "LROP",
+            "category": "arrival",
+            "route": "SOMOV.SORDU1U.LROP",
+            "altitude": [14000, 17000],
+            "speed": 350,
```

Comment:
This `speed` may be too fast for the aircraft in the `ROT` fleet, such as the AT45. 320 should do fine!